release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -157,6 +157,15 @@ For more information about this feature on bare-metal infrastructure, see xref:.
 [id="ocp-4-16-storage"]
 === Storage
 
+[id="ocp-4-16-storage-secrets-store-csi-driver-vault"]
+==== HashiCorp Vault is now available for the {secrets-store-operator}
+
+You can now use the {secrets-store-operator} to mount secrets from HashiCorp Vault. The {secrets-store-operator} is available as a Technology Preview feature.
```

Comment:
Should it be mentioned here that users can use this Operator to mount secrets from HashiCorp Vault -- to a CSI volume in {product-title}.